cdap-ui/app/directives/epoch/charts.js

```diff
@@ -72,7 +72,7 @@ ngEpoch.controller('epochController', function ($scope, $compile, caskWindowMana
         }
         else if (newVal && newVal.length) {
           for (var i = 0; i < newVal.length; i++) {
-            $scope.me.push([newVal[i]]);
+            $scope.me.push(newVal[i]);
```

Comment:
What happens if I don't get want multiple metrics? In the case of cdap dashboard? This throws me an error in the console when I am in the cdap (standard) dashboard